TestMod/TestMod.cs

```diff
@@ -482,7 +482,7 @@ private void AddEmptyItems()
 
         // Add new items as copies of vanilla items - just works when vanilla prefabs are already loaded (ObjectDB.CopyOtherDB for example)
         // You can use the Cache of the PrefabManager in here
-        private void AddClonedItems(On.ObjectDB.orig_CopyOtherDB orig, ObjectDB self, ObjectDB other)
+        private void AddClonedItems()
         {
             // You want that to run only once, Jotunn has the item cached for the game session
             if (!clonedItemsProcessed)
```

Comment:
i would change that according to the event comment. executy once and unsubscribe after that. you want to implement, ideath?